docs/pudl/pudl-etl.dot

```diff
@@ -0,0 +1,184 @@
+digraph pudl_etl {
```

Comment:
what is this file?

Reply:
It's a machine readable specification for a flowchart outlining the ETL process. I created it a while ago and didn't want to lose that work. It's not integrated into the docs currently.